tools/nitrogfx/main.c

```diff
@@ -381,31 +381,31 @@ void HandlePngToGbaCommand(char *inputPath, char *outputPath, int argc, char **a
             if (options.numTiles < 1)
                 FATAL_ERROR("Number of tiles must be positive.\n");
         }
-        else if (strcmp(option, "-mwidth") == 0)
+        else if (strcmp(option, "-mwidth") == 0 || strcmp(option, "-cpc") == 0)
         {
             if (i + 1 >= argc)
-                FATAL_ERROR("No metatile width value following \"-mwidth\".\n");
+                FATAL_ERROR("No columns per chunk value following \"%s\".\n", option);
```

Comment:
As an uninformed normie, "chunks" doesn't inherently make sense to me, which would be fine if it weren't for the fact that we could get people asking about what this error message means. Maybe I'm wrong and it makes sense tho

Reply:
a chunk is a collection of tiles
columns per chunk is how many columns of tiles there are per chunk
this used to be metatile but that was *more* confusing, cause metatile means... nothing really

Reply:
nitro studio documentation calls them cells and multicells, that work?

Reply:
cells are another thing, don't want the same term for two different separate things

Reply:
eh can't think of anything better so it's prolly fine